### PR TITLE
create kubelet client cert with correct group

### PR DESCRIPTION
**Pull request by deads2k**
Uses cfssl to create a `kubelet.crt`.

### CODE REVIEW

**Comment by deads2k** [comment]
@k8s-bot cvm gce e2e test this

**Comment by deads2k** [comment]
Looks like cfssl isn't available.  

```
I1220 07:12:26.616] /workspace/kubernetes/cluster/../cluster/../cluster/gce/../../cluster/gce/../../cluster/common.sh: line 943: cfssl: command not found
W1220 07:12:26.616] + echo '{"signing":{"default":{"expiry":"43800h","usages":["signing","key encipherment","client auth"]}}}'
I1220 07:12:26.616] /workspace/kubernetes/cluster/../cluster/../cluster/gce/../../cluster/gce/../../cluster/common.sh: line 943: cfssljson: command not found
```

@mikedanese @cjcullen you were expecting cfssl to be available, right?

**Comment by deads2k** [comment]
I think this is ready for review.  It ought to clean up the RBAC enablement nicely.

File: cluster/common.sh [hunk 1]
```diff
@@ -932,9 +932,20 @@ function generate-certs {
...
+
+    download-cfssl
+
+    # make the config for the signer
+    echo '{"signing":{"default":{"expiry":"43800h","usages":["signing","key encipherment","client auth"]}}}' > "ca-config.json"
+    # create the kubelet client cert with the correct groups
+    echo '{"CN":"kubelet","names":[{"O":"system:nodes"}],"hosts":[""],"key":{"algo":"rsa","size":2048}}' | "${KUBE_TEMP}/cfssl/cfssl" gencert -ca=pki/ca.crt -ca-key=pki/private/ca.key -config=ca-config.json - | "${KUBE_TEMP}/cfssl/cfssljson" -bare kubelet
+    mv "kubelet-key.pem" "pki/private/kubelet.key"
+    mv "kubelet.pem" "pki/issued/kubelet.crt"
+    rm -f "kubelet.csr"
```
Comment (mikedanese) [inline cluster/common.sh]: Do we delete csr's created with easy rsa? Doesn't kube_temp get deleted all the same later on?

**Comment by mikedanese** [comment]
looks fine.

**Comment by zmerlynn** [comment]
This change broke the (deprecated) `kube-up.sh` deployment for AWS.

File: cluster/gce/util.sh [hunk 2]
```diff
@@ -726,29 +726,12 @@ function get-master-disk-size() {
...
-#   ETCD_CA_CERT_BASE64
-#   ETCD_PEER_KEY_BASE64
-#   ETCD_PEER_CERT_BASE64
-#
-function create-etcd-certs {
-  local host=${1}
-  local ca_cert=${2:-}
-  local ca_key=${3:-}
-
+function download-cfssl {
```
Comment (zmerlynn) [inline cluster/gce/util.sh]: Is there a reason this function isn't in `cluster/common.sh` vs here?

**Pull request merged**